.gitlab-ci.yml

```diff
@@ -0,0 +1,62 @@
+stages:
+  - test
+  - build
+variables:
+  GIT_DEPTH: "3"
+  SIMPLECOV: "true"
+  RUST_BACKTRACE: "1"
+  RUSTFLAGS: ""
+  CARGOFLAGS: ""
+cache:
+  untracked: true
+linux-test:
+  stage: test
+  image: node:8
+  only:
+    - beta
+    - tags
+    - stable
+    - triggers
+    - master
+  script:
+    - curl -o- -L https://yarnpkg.com/install.sh | bash 
+    - export PATH=$HOME/.yarn/bin:$PATH
+    - yarn install
+    - yarn lint
```

Comment:
I don't think we want to include `yarn lint`. `yarn test` already checks if the files are correctly linted

Reply:
Thanks will remove.